source/addonHandler/__init__.py

```diff
@@ -44,6 +45,12 @@
 _blockedAddons=set()
 
 
+# Allows add-ons to process additional command line arguments when NVDA starts.
+# Each handler is called with one keywod argument `cliArgument`
+# and should return `False` if it is not interested in it, `True` otherwise.
+isCLIParamKnown = extensionPoints.AccumulatingDecider(defaultDecision=False)

```

Comment:
Could you add an example of how an addon may use this extension point to the documentation here. 

For example, something like this:
```py
def processArgs(cliArgument: str) -> bool:
     if cliArgument == "enable-example-addon-feature":
            doSomething()
            return True
     return False

isCLIParamKnown.register(processArgs)
```